test/src/examples_complete_test.go

```diff
@@ -3,32 +3,41 @@ package test
 import (
 	"math/rand"
 	"strconv"
+	"strings"
 	"testing"
 
 	"github.com/gruntwork-io/terratest/modules/terraform"
 	"github.com/stretchr/testify/assert"
 )
 
+func listContains(s []string, e string) bool {
+	for _, a := range s {
+		if a == e || strings.Contains(a, e) {
+			return true
+		}
+	}
+	return false
+}
+
 // Test the Terraform module in examples/complete using Terratest.
 func TestExamplesComplete(t *testing.T) {
 	t.Parallel()
 
-	randId := strconv.Itoa(rand.Intn(100000))
-	attributes := []string{randId}
-
-	exampleInput := "Hello, world!"
+	// We always include a random attribute so that parallel tests and AWS resources do not interfere with each
+	// other
+	randID := strconv.Itoa(rand.Intn(100000))
+	attributes := []string{randID}
 
 	terraformOptions := &terraform.Options{
 		// The path to where our Terraform code is located
 		TerraformDir: "../../examples/complete",
 		Upgrade:      true,
 		// Variables to pass to our Terraform code using -var-file options
 		VarFiles: []string{"fixtures.us-east-2.tfvars"},
-		// We always include a random attribute so that parallel tests
-		// and AWS resources do not interfere with each other
 		Vars: map[string]interface{}{
-			"attributes": attributes,
-			"example":    exampleInput,
+			"attributes":        attributes,
+			"create_sns_topic":  true,
+			"enabled_standards": []string{"standards/aws-foundational-security-best-practices/v/1.0.0", "ruleset/cis-aws-foundations-benchmark/v/1.2.0"},
```

Comment:
we need to move all the configs to examples/complete/fixtures.us-east-2.tfvars 
The terratest should not alter any configurations (except for the random attribute to make all names unique on each run)

Reply:
How does that work with assertions? Do we just assume the proper values will be set/not modified in the fixture?

For example, I'm asserting that the SNS topic is created, but obviously, that will fail if someone changes `create_sns_topic=false` in the fixture.

Reply:
but the test is for the example.
we just have that pattern to put all the configs in `examples/complete/fixtures.us-east-2.tfvars`, and point the test to it.

Reply:
ed688ad06a8850ef65b6833b1549b5a780e9e69f

Reply:
This is fine for now, but to @mcalhoun point we should support testing multiple feature flags, but have avoided that for now